[Vectorize] Get started pages minor fixes (#11823)
cloudflare/cloudflare-docs: content/vectorize/get-started/intro.md

## 4. Insert vectors

> Before you can query a vector database, you need to insert vectors for it to query against. These vectors would be generated from data (text, images, etc) you pass to a machine learning model. However, this tutorial will define static vectors to illustrate how vector search works on its own.

Reviewer: ```suggestion
Before you can query a vector database, you need to insert vectors for it to query against. These vectors would be generated from data (such as text or images) you pass to a machine learning model. However, this tutorial will define static vectors to illustrate how vector search works on its own.
```
Avoiding "etc".

```sh
$ npx wrangler deploy
```
Once deployed, preview your Worker at `https://vectorize-tutorial.<YOUR_SUBDOMAIN>.workers.dev`.

## 7. Query your index

> To insert vectors and then query them, use the URL for your deployed Worker, such as `https://vectorize-tutorial.<YOUR_SUBDOMAIN>.workers.dev/`, open your browser and:

Reviewer: ```suggestion
To insert vectors and then query them, use the URL for your deployed Worker, such as `https://vectorize-tutorial.<YOUR_SUBDOMAIN>.workers.dev/`. Open your browser and:
```

1. Insert your vectors first by visiting `/insert`. This should return the below JSON:

```json
// https://vectorize-tutorial.<YOUR_SUBDOMAIN>.workers.dev/insert
{ "count": 5, "ids": [1, 2, 3, 4, 5] }
```

Subsequent visits will return `count:0` as you cannot `insert()` the same vector IDs.

> 2. Query your index - expect your query vector of `[54.8, 5.5, 3.1]` to be closest to vector ID `5` by visiting the root path of `/` . This will return the three (`topK: 3`) closest matches, as well as their vector values and metadata.

Reviewer: ```suggestion
2. Query your index - expect your query vector of `[54.8, 5.5, 3.1]` to be closest to vector ID `5` by visiting the root path of `/` . This query will return the three (`topK: 3`) closest matches, as well as their vector values and metadata.
```
